Autopilot/SensorFusion/Src/SensorFusion.cpp

```diff
@@ -126,7 +126,7 @@ double * cartesianToGPS(double x, double y){
     return latLong;
 }
 
-SFError_t SF_GetAttitude(SFAttitudeOutput_t *Output, IMUData_t *imudata, airspeedData_t *airspeeddata) {
+SFError_t SF_GetAttitude(SFAttitudeOutput_t *Output, IMUData_t *imudata) {
```

Comment:
There a reason we are remving the airspeed data? Or is it not needed anymore

Reply:
The original attitude function included airspeed but never actually used the data, so it's not needed.

Reply:
sounds good